### ISSUE TITLE
Work Tracker

### ISSUE THREAD

**Issue by heyjay44** [fireproof-storage/fireproof#351]
### Sprint Goals for December 2nd
- Meno: 
      - working on the cloud (0.19) and it is done but it testing (Weds). 
      - Cloud auth conversation
      - esm problem - significant
      - tests not passing right now
- Steven: 
      - 
- Marty: 
      -  issue with useLiveQuery 

Other tasks:
- Jen to merge: https://github.com/fireproof-storage/fireproof/pull/401
- https://fireproof.storage/   codepen --- move from <script src=....> to import @jchris 

Decisions from this week: 
1. UCANs are not the priority at the moment. 

Things to tackle:
1. Get netlify over the line
2. Everyone to review: https://www.notion.so/How-We-Work-1539e5107ccb80b8b8bbe297698e2506

### **App Updates & Development**
- [x] @mschoch updated his to do app for 0.19
- [ ] updating https://github.com/fireproof-storage/catbot @valorant-dhruv 
      - Ran into some issues. Marty helped here: https://github.com/valorant-dhruv/fireproof-catbot-v2/pull/1
- [ ] Update Firehouse app @mschoch Goal: CF AI Agent Collab @threepointone
        - [ ] working app on 0.19 with any relevant updates reflected in the docs @mschoch 
        - [x] https://github.com/fireproof-storage/fireproof/issues/349 @mabels can help, but we will save for later since party kit is out of scope
        - [ ] Taking out the AI portion, and focus on the firehouse chat. (Old version of this app used partykit). 
        - [ ] Linking fireproof cloud to party kit AI backend

### **Tests**
- [ ] test suite - focus on fireproof cloud. Current situation: it is currently commented out. Meno discovered that the cloud implementation breaks if the test suite runs against it. @jchris to unblock @mabels  
Current work: @mabels is working on 0.20, did work on 0.19. 

### **Cloud**
- [ ] Building the 0.19 cloud backend to get it working. @mabels 

### **Docs**
- [x] we need to locate all docs, triage based on priorities @mschoch 
- [ ] we need to make a note in the docs, change database to ledger and make a note at the top of the docs stating that database is the old name of ledger (which some ppl might bump into). @jchris provide the PR#14
- [ ] Docs housekeeping and improvement: https://github.com/fireproof-storage/fp-website/issues/42
- [ ] @jchris @jnsnco to discuss optimising docs

### **Sign in and user growth**
- **Reference material**: [Mikeal's comments](https://gist.github.com/mikeal/567f1488e8f68390bc32096a7fa95546 )& [JChris' Auth flow](https://www.tldraw.com/r/M7tmHj6hE-i3ILLQhkVh8?d=v-768.-246.5000.2830.page)
- [x] Move github sign up flow to fireproof's github
- [ ] https://github.com/fireproof-storage/dashboard/issues/17 @necrodome @meghansinnott @AnnaFireproof 

### **Partnerships** ###
Goal: Netlify co-marketing by end of 2024
- [ ] Netlify @mschoch 
      - [ ] now public, but unlisted in the directory
      - [ ] we need to validate that it works 
      - [ ] @jchris submitting a form, but only after @mschoch says "go"

### **Dashboard**
- **Reference Material for Dashboard Work**: [Updated dashboard](https://www.tldraw.com/s/v2_c_Qp9MeyvYUei87kc0d3C_d?d=v-44.-207.2292.1297.7RMeJAYs3lFU13kHBxm8L)
- [x] dashboard homepage: some messaging about "being in the right place" and some documentation about how to add to your code
- [ ] resolve expiring links @mabels 
- [ ] brainstorm a plan for UX testing by people unfamiliar with Fireproof @meghansinnott 

### **UCAN**
- We need a bigger plan on UCAN
Need: background on UCAN? Fireproof blog post. Self-sovereign auth with UCAN and Web Crypto
- [ ] Take over ucan.xyz @meghansinnott 
- @mabels to jump to UCAN after cloud (now without UCAN)
- off launch list - ucan butler - referenced in the tldraw - using Steven's work as a template. <- makes a great newsletterveryone

**Comment by jchris** [comment]
@mikeal is update the edit view
